cli/mender-artifact/main.go

```diff
@@ -229,6 +229,14 @@ func run() error {
 		},
 	}
 
+	globalFlags := []cli.Flag{
+		cli.StringFlag{
+			Name:  "compression",
+			Value: "gzip",
+			Usage: "Compression to use for data and header inside the artifact.",
+		},
```

Comment:
I think the available choices should be listed here. I assume this is "gzip" and "none"?